### ISSUE TITLE
JSON TypeError in Import

### ISSUE THREAD

**Issue by Canuck317** [CTFd/CTFd#1060]
**Environment**:

  - CTFd Version/Commit: 2.1.3 (Last pull was yesterday)
  - Operating System: Debian 9 'Stretch' (apt update && apt upgrade yesterday)
  - Web Browser and Version: Mozilla Firefox 60.8.0esr 64-bit
  - Database Version: MariaDB 10.3.16
  - Python 2.7.13 and 3.5.3
  - gunicorn 19.9.0
  - Running CTFd using gunicorn and gevent workers.
  - Database name: testing
  - Reproduced same issue on openSUSE box with MariaDB 10.2.22

**What happened?**

On attempting a database import using Python 3 I get error "TypeError("the JSON object must be str, not 'bytes'",)" Any attempts to access CTFd in the web browser then results in 500 Internal Server Error. I have attached the gunicorn output, there appears to be an error in importing the flags table? (Flags are a mix of text, or sha1 hashes)

(Also an odd error with both Python 2 and 3 where starting with an empty database gives a duplicate challenge table error, but restarting the program seems to work past that. Attached error log just for the heck of it. NOTE: Nevermind, this appears to be from having 4 workers running when starting with an empty database.)

**What did you expect to happen?**

I expected the database to import successfully on both versions of Python. (This does happen when using Python 2.)

**How to reproduce your issue**

I carried out the following steps:

Successful import with Python 2
- Drop & recreate database in mysql:
  - drop database testing;
  - create database testing;
  - grant all privileges on testing.* to 'ctfd'@'localhost';
  - flush privileges;
- Delete CTFd/.data folder
- Ensure we're using the Python 2 version of gunicorn:
  - sudo -H pip3 uninstall gunicorn   (note prompt for removal of 3.5 supporting packages)
  - sudo -H pip install gunicorn
- Start CTFd
  - gunicorn --bind 0.0.0.0:8000 -w 4 --graceful-timeout 12 -k gevent "CTFd:create_app()"
- Create temp setup, import previously exported database.  
- EVERYTHING WORKS.

Failed import with Python 3
- Drop & recreate database, same as above.
- Delete CTFd/.data folder
- Ensure we're using the Python 3 version of gunicorn:
  - sudo -H pip uninstall gunicorn   (note prompt for removal of 2.7 supporting packages)
  - sudo -H pip3 install gunicorn 
- Start CTFd, same command as above.
- Create temp setup, import previously exported database.
- JSON Error as above.
[CTFd_Imprt_Error_Gunicorn_Output.txt]

**Any associated stack traces or error logs**
CTFd_Import_Error_Gunicorn_Output.txt shows the error in import.
(https://github.com/CTFd/CTFd/files/3397460/CTFd_Imprt_Error_Gunicorn_Output.txt)
CTFd_Duplicate_Challenges_Table.txt - Odd error on first start.
[CTFd_Duplicate_Challenges_Table.txt](https://github.com/CTFd/CTFd/files/3397461/CTFd_Duplicate_Challenges_Table.txt)

**Comment by Canuck317** [comment]
Small update:

After noticing the log showed the JSON error on hitting the flags table, I went to my production database and went through all my flags be sure I didn't have any static hash flags left.  They're now all text, or regex ith ^.*(<hash>).*$. (So students can enter the hash, or FLAG-HASH, which the challenges usually spit out.)

I then exported my database again using both Python 2 and Python 3 versions of gunicorn. I did receive a polymorphic identity error in the gunicorn output when outputting the flags table when running Python 2 (attached).  This error did not appear with Python 3.

However, there was no change in the behaviour of my testing setup.  Both exports failed to import when running CTFd with Python 3 gunicorn. Both imported successfully when running CTFd with Python 2 gunicorn. Same error as before.

[CTFd_Polymorphic_identity_error_python2.txt](https://github.com/CTFd/CTFd/files/3397728/CTFd_Polymorphic_identity_error_python2.txt)
